projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -499,27 +499,10 @@ function is_visible() {
 
```

Comment:
Similar to `class.wpcom-json-api-update-site-logo-endpoint.php`, I'm not sure exactly how this is used, so help figuring out how to test is appreciated.

Reply:
Similarly, this response is also covered by wpcom tests.